examples/vanilla-js/app.js

```diff
@@ -0,0 +1,80 @@
+import { ContentfulLivePreview } from '@contentful/live-preview';
+import { createClient } from 'contentful';
+import dotenv from 'dotenv';
+
+// Configuration object
+const CONFIG = {
+  locale: 'en-US',
+  entryId: 'W9z8t0uXz3CMFjp4sw5zj',
+  fields: ['title'],
+  subscriptions: [],
+  debugMode: true,
+};
+
+document.addEventListener('DOMContentLoaded', initialize);
+
+function initialize() {
+  dotenv.config();
+
+  const client = createClient({
+    space: process.env.CONTENTFUL_SPACE_ID,
+    accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+    host: 'preview.contentful.com',
+  });
+
+  ContentfulLivePreview.init({ locale: CONFIG.locale });
+
+  client
+    .getEntry(CONFIG.entryId)
+    .then((entry) => {
+      CONFIG.fields.forEach((fieldId) => {
+        displayFieldData(entry, client, fieldId);
+        setupLivePreview(entry, fieldId);
+      });
+    })
+    .catch((err) => console.error(err));
+}
+
+function setAttributes(element, attributes) {
+  for (const [key, value] of Object.entries(attributes)) {
+    // Skip undefined or null values
+    if (value === undefined || value === null) continue;
+
+    if (key.startsWith('data-')) {
+      const formattedName = key.split('data-')[1].replace(/-([a-z])/g, (m, w) => w.toUpperCase());
+      element.dataset[formattedName] = value;
+    } else {
+      element.setAttribute(key, value);
+    }
+  }
+}
+
+function displayFieldData(entry, client, fieldId) {
```

Comment:
Not sure about this one, or if we should then create the attributes manually in the HTML file, see https://github.com/contentful/live-preview/pull/287/files#r1303901947

Reply:
We had it like that in the documentation before, but I am also happy to go with the manual approach. I'll change it in both places then

Reply:
I think in the documentation it's fine if we mention the function - for example, if you have a node server and render the HTML on the server, you could use the helper to create the tags

Reply:
I'v changed it to set the attributes manually in the HTML. Does it look good now?